### ISSUE TITLE
[TypeScript] `DeviceCapability.setValue` type error (`opts` not optional)

### ISSUE THREAD

**Issue by OlivierZal** [athombv/web-api-issues#27]
Since `opts` means `options`, this parameter should be optional, so 

```
export class DeviceCapability  {
  value: boolean | number | string | null;
  lastChanged: Date | null;
  destroy(): any;
  setValue(
    value: boolean | number | string,
    opts: object,
  ): Promise<any>;
}
```

should rather be:

```
export class DeviceCapability  {
  value: boolean | number | string | null;
  lastChanged: Date | null;
  destroy(): any;
  setValue(
    value: boolean | number | string,
    opts?: object,
  ): Promise<any>;
}
```

**Comment by jeroenwienk** [comment]
Fixed in 3.0.7

**Comment by OlivierZal** [comment]
Confirmed
